modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -236,7 +242,7 @@ export SPACEWALK_USER='<adminname>';export SPACEWALK_PASS='<password>'
 . Set up a cron job to run the script daily:
 +
 ----
-ln -s /usr/local/bin/cent-errata.sh /etc/cron.daily
+mgrctl exec -- ln -s /usr/local/bin/cent-errata.sh /etc/cron.daily
```

Comment:
Adding cron files in the container won't fly. However the `errata-import.pl` script that the job runs uses the XML-RPC API and could run on the host.

In this case we should just document to add this to the host and not in the container.

This means that the script file shouldn't be created in the container but in the host.